ReDress/Main.cs

```diff
@@ -294,7 +294,8 @@ static void OnGUI(UnityModManager.ModEntry modEntry) {
                                                     EntityPartStorage.perSave.IncludeByName[pickedUnit.UniqueId] = tmpIncludes;
                                                     EntityPartStorage.SavePerSaveSettings();
                                                 }
-                                                GUILayout.TextArea($"    {eeName}", GUILayout.ExpandWidth(false));
+                                                string itemName = settings.AssetIds.Where(t => t.Item1 == eeName).Select(t => t.Item2).FirstOrDefault();
+                                                GUILayout.Label($"    {itemName}", GUILayout.ExpandWidth(false));
```

Comment:
Why did I use TextArea previously? Maybe Label doesn't allow selecting + copying, but TextArea does?

Also, it's probably better to do {itemName} ({eeName}?

Reply:
That’s probably why. I can do both for users and developers and recommit.